src/ResponseFactory.php

```diff
@@ -105,14 +105,18 @@ public function render(string $component, $props = []): Response
     /**
      * @param string|RedirectResponse $url
      */
-    public function location($url): \Symfony\Component\HttpFoundation\Response
+    public function location($url, ?array $headers = []): \Symfony\Component\HttpFoundation\Response
```

Comment:
This doesn't need to be nullable since a default is set:

```suggestion
    public function location($url, array $headers = []): \Symfony\Component\HttpFoundation\Response
```